spyder/plugins/maininterpreter/widgets/tests/test_status.py

```diff
@@ -19,8 +19,13 @@
 from spyder.config.base import running_in_ci
 from spyder.plugins.statusbar.widgets.tests.test_status import status_bar
 from spyder.plugins.maininterpreter.widgets.status import InterpreterStatus
+from spyder.utils.conda import get_list_conda_envs
 
 
+CONDA_MISSING = (len(get_list_conda_envs()) == 0)
```

Comment:
We have a function called `is_anaconda` in `spyder.config.utils`. So please use that one instead.

Also, there's no need to declare an extra constant here for that.

Reply:
Ah, I didn't know about that function, thanks!
There's a newer comment on [the stackoverflow thread](https://stackoverflow.com/a/47610844/3889398), by the way, which suggests that `is_anaconda` should use `sys.base_prefix` instead of `sys.prefix` for greater robustness.